content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -82,20 +82,22 @@ to `false`. For example:
    kubectl attach -it userns bash
    ```
 
-And run the command. The output is similar to this:
+And run these commands, the output is similar to this:
 
 ```none
-readlink /proc/self/ns/user
+$ readlink /proc/self/ns/user
```

Comment:
We [don't include the command prompt](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt), especially not in sample output.

Reply:
Fixed